srcopsmetrics/kebechet_metrics.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (C) 2021 Dominik Tuchyna
+#
+# This file is part of thoth-station/mi - Meta-information Indicators.
+#
+# thoth-station/mi - Meta-information Indicators is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# thoth-station/mi - Meta-information Indicators is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with thoth-station/mi - Meta-information Indicators.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Kebechet repository metrics evaluation."""
+
+import logging
+import os
+import time
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+import numpy as np
+import pandas as pd
+from github import Github
+
+from srcopsmetrics.entities.issue import Issue
+from srcopsmetrics.entities.pull_request import PullRequest
+from srcopsmetrics.storage import KnowledgeStorage
+
+BOT_NAMES = {"sesheta"}
+
+UPDATE_TYPES_AND_KEYWORDS = {
+    "automatic": "Automatic update of dependency",
+    "failure_notification": "Failed to update dependencies to their latest version",
+    "initial_lock": "Initial dependency lock",
+}
+
+_LOGGER = logging.getLogger(__name__)
+_GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
+
+
+class KebechetMetrics:
```

Comment:
do we want to call it `KebechetUpdateManagerMetrics` or have a base class `KebechetMetrics`, to be reused for the different managers? wdyt?

Reply:
I would stick to the `KebechetMetrics` as  a base class, wihitn which the other managers would be calculated

Reply:
@pacospace look here https://github.com/thoth-station/mi/pull/332/files#r578480779

Reply:
so I think it should be currently only one `KebechetMetrics` class, because the `prs` and `issues` are shared across all the other performance analysis, so I guess it's pretty handy to have it under one class rn.